Makefile

```diff
@@ -37,14 +37,9 @@ CPPFLAGS ?=
 # Linker flags
 LDFLAGS ?=
 
-##############################################################################
-
-# Update on each new release!!
-RELEASE_VERSION = 0.1.0
-
 # Pass the version to the command line program (pulled from tags).
-TAG_VERSION = $(shell git describe --tags 2>/dev/null)
-VERSION = $(if $(TAG_VERSION),$(TAG_VERSION),$(RELEASE_VERSION))
+VERSION ?= $(shell git describe --tags 2>/dev/null)
+CPPFLAGS += $(if $(VERSION),-DVERSION="\"$(VERSION)\"")
```

Comment:
I think this should be `override CPPFLAGS +=` instead of just `CPPFLAGS +=`.  Otherwise if someone specifies `make CPPFLAGS=...` on the command line, `-DVERSION` won't be added.

Reply:
Added `override`.